Xamarin.PropertyEditing/ViewModels/BrushPropertyViewModel.cs

```diff
@@ -47,7 +48,11 @@ public BrushPropertyViewModel (TargetPlatform platform, IPropertyInfo property,
 		protected override async Task UpdateCurrentValueAsync ()
 		{
 			await base.UpdateCurrentValueAsync ();
+			if (Solid != null) {
+				Solid.Reset();
+			}
 			OnPropertyChanged (nameof (Opacity));
+			OnPropertyChanged (nameof (Solid));
```

Comment:
This is incorrect, `Solid` has not changed, it stays the same value from `ctor`. This will force all `Solid.*` bindings to re-query after they've already requeried the individual properties that update from `Reset`.